### PR TITLE
Change links from readthedocs.org to readthedocs.io.

### PR DESCRIPTION
**Pull request by Natim**
r? @leplatrem

### CODE REVIEW

File: docs/configuration/settings.rst [hunk 1]
```diff
@@ -627,7 +627,7 @@ Project information
 +=======================================+============================================+==========================================================================+
 | kinto.error_info_link                 | ``https://github.com/kinto/kinto/issues/`` | The HTTP link returned when uncaught errors are triggered on the server. |
 +---------------------------------------+--------------------------------------------+--------------------------------------------------------------------------+
-| kinto.project_docs                    | ``http://kinto.readthedocs.org``           | The URL where the documentation of the Kinto instance can be found. Will |
+| kinto.project_docs                    | ``http://kinto.readthedocs.io``           | The URL where the documentation of the Kinto instance can be found. Will |
```
Comment (leplatrem) [inline docs/configuration/settings.rst]: the vertical bars might deserve some horizontal adjustment

**Comment by leplatrem** [comment]
r+wc

File: docs/configuration/settings.rst (hunk 1 above)
Comment (Natim) [inline docs/configuration/settings.rst]: Good catch

**Comment by almet** [comment]
This makes me wonder: should we host these docs under docs.kinto-storage.org instead?

**Comment by Natim** [comment]
I don't think so.

**Pull request merged**